### ISSUE TITLE
Run as service for Windows (NSSM)

### ISSUE THREAD

**Issue by ft-scobra** [Beannsss/PlexAniSync#20]
Hey. This script really saved me from regretting switching over to Emby from Jellyfin. However I noticed that there isn't a guide to run the script as a windows service.
I got it working with [nssm](https://nssm.cc/) and I was wondering if I can add that to the readme. Meaning, I'll add a few line on how to set it up.

Windows Services:
![image](https://github.com/user-attachments/assets/217f6975-9d42-44fa-8be3-8c268c6012cc)

Logs:
![image](https://github.com/user-attachments/assets/2de414bd-2652-40e8-aa7c-e7c7b89a9e63)

**Comment by bpwhelan** [comment]
Yeah that would be great. Any specific reason you went with nssm? Like is there anything that stops this from going through the normal manager? I have pretty much 0 experience with windows services since everything i do personally and professionally is on linux, but im a bit weary about adding instructions for something that doesn't ship with Windows.

**Comment by ft-scobra** [comment]
I just wanted it to run in the background without any terminal. I'm only a beginner in programming so the way I was using this was opening cmd and `python EmbyAniSync.py`. But I didn't want the terminal so I searched for a way to run a python script as a service. nssm just seemed like the most common answer and it was pretty easy to setup. 

As for the reason I'm running it on Windows, Like I said, I'm still new to programming and it's the only device I have for now lol.

**Comment by ft-scobra** [comment]
Also, this was what I was using for Jellyfin.
https://github.com/vosmiic/jellyfin-ani-sync

It was just a plugin that was easy to setup within Jellyfin. Is there anyway this script could be edited to be something similar? Since Emby has plugins.

**Comment by bpwhelan** [comment]
I was not aware of this plugin when I wrote EmbyAniSync to be completely honest. My script would have to be entirely re-written in C# for it to be a plugin.

However, I wanted to make EmbyAniSync a plugin at some point, so I may look at this project and adapt it for Emby.

**Comment by bpwhelan** [comment]
No promises, but forked that repo, you can see my progress (or lack of) here https://github.com/bpwhelan/Emby.Anime.Sync

I have never touched C# so I might not even get anywhere with it though.

**Comment by ft-scobra** [comment]
Same here. I have 0 experience in C#. Will definitely consider contributing if I ever learn C# in the future though.

**Comment by bpwhelan** [comment]
The more I play around with trying to adapt this code to Emby, the less confident I am that I will be able to do it, Maybe in the future will I think about starting something from scratch, but that won't be for a very long time I don't think.

**Comment by ft-scobra** [comment]
No pressure. I just wanted to share that since I was using it before. 

Have been using EmbyAniSync as a service (on Windows) without any major issues so far. The only minor issue is that it takes in the number of episodes rather than the the episode number. Meaning, if I have only 5/24 episodes downloaded, but those are EP15-19 then it won't count as EP19 but as EP5. Not sure if this is intended but I'm pretty sure there should be a way to use the episode number instead yes?

For context:
- I have 5 Episodes downloaded for Slime Season 3.
![image](https://github.com/user-attachments/assets/080bee2e-64a4-4993-8465-846e9dba4abe)
- Script detects as EP5 instead of EP19. 
![image](https://github.com/user-attachments/assets/affeaf04-7bae-4502-92f5-cd457872ddf1)

**Comment by ft-scobra** [comment]
Just realized, I should've added that as a separate Issue.

**Comment by bpwhelan** [comment]
You can open an issue yes, but the likelihood of fixing is a bit low, it complicates things a bit
